ServerSocketTest/src/main/Main.java

```diff
@@ -50,38 +74,123 @@ public static void main(String[] args) {
 				
 				
 				//get the first line of the request
-				String firstLine=request.toString().split("\n")[0];
+				String firstLine=header.toString().split("\n")[0];
 				//System.out.println(firstLine);
 				//get the second element from first line
 				String resource=firstLine.split(" ")[1];
 				String method=firstLine.split(" ")[0];
 				
-				System.out.println("Method: "+method);
-				
-				System.out.println("Resource: "+resource);
+
 				//compare the element to our list of things
 				//send back apprpiate thing
-				if(resource.equals("/people"))
+				
+				/*when the method in the header is a GET we retrieve the resource. 
+				 * the GET method may have an optional body but we ignore for now 
+				*/
+				if(method.equals("GET"))
 				{
-					OutputStream clientOutput=client.getOutputStream();//output all responses
-					
-					clientOutput.write(("HTTP/1.1 200 OK\r\n").getBytes());//encode to bytes
-					clientOutput.write(("\r\n").getBytes());//blank line
-					clientOutput.write(("Mark \r\nDorian \r\nMehir \r\nDan \r\nKhate \r\nRobert \r\nCiaran \r\nDillon").getBytes());//encode to bytes
+					if(resource.equals("/people"))//check the resource we are looking for
+					{
+						OutputStream clientOutput=client.getOutputStream();//output all responses
+						
+						clientOutput.write(("HTTP/1.1 200 OK\r\n").getBytes());//encode to bytes
+						clientOutput.write(("\r\n").getBytes());//blank line
+						//people are contained in an arraylist. So when queried we output everyone in the list
+						for(int i=0;i<people.size();i++)
+						{
+							clientOutput.write((people.get(i)+" \r\n").getBytes());//encode to bytes
+							
+						}
 				
-					clientOutput.flush();//empty the built up buffer
+						
+						clientOutput.flush();//empty the built up buffer
+					}
+					else if(resource.equals("/"))//the root doesnt do anything rn. Just a "welcome page"
+					{
+						OutputStream clientOutput=client.getOutputStream();//output all responses
+						
+						clientOutput.write(("HTTP/1.1 200 OK\r\n").getBytes());//encode to bytes
+						clientOutput.write(("\r\n").getBytes());//blank line
+						clientOutput.write(("Hello").getBytes());//encode to bytes
```

Comment:
What if we want to pull multiple requests from the server. Would we have to do .write(("HTTP/1.1 200 OK\r\n").getBytes() and .write(("\r\n").getBytes()  for every message we want to pull or just once at the start.

Reply:
Each request has a response. So I would say yes to this.

Reply:
Just to clarify we will wrap this in a method and just call the method every time we have a 200 OK response